adaptive_navigation/lib/src/adaptive_app_bar.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// AdaptiveAppBar has a leading width of 72.0. Everything else is the same as
+/// [AppBar].
+class AdaptiveAppBar extends AppBar {
```

Comment:
The Adaptive App Bar goes changes width size depending on screen size

Reply:
I agree that it is a bit overkill. A better solution might deserve more investigation.Maybe leadingWidth param can be added to the Theme in flutter/flutter? 

Regardless, when deciding what leadingWidth to use, it would need to know what NavigationType is being used. NavigationType.rail can be configured for any size (medium, large, etc), but here the 72 width is only being applied to a medium screen, which might not have a navigation rail at all.